src/Tests/ObjectLifetimeTests/Tests/ObjectLifetimeTests.cs

```diff
@@ -0,0 +1,1483 @@
+
+                    ClrObject2 _clrObj = new ClrObject2();
+
+                    BindingOperations.SetBinding(stackPanel1, FrameworkElement.TagProperty, new Binding() { Source = _clrObj, Path = new PropertyPath("IntProperty") });
+                    BindingOperations.SetBinding(stackPanel2, FrameworkElement.TagProperty, new Binding() { Source = _clrObj, Path = new PropertyPath("IntProperty") });
+
+                    // Get weak references to the managed objects so that we can monitor them.
+                    _element1Ref = new WeakReference(_element1);
+                    _element2Ref = new WeakReference(_element2);
+                    _clrObjRef = new WeakReference(_clrObj);
+                })
+                .CallFromUIThread(() =>
+                {
+                    // Verify all objects are still alive.
+                    GC_ForceCollect();
+
+                    Verify(_element1Ref.IsAlive, "Element 1 died");
+                    Verify(_element2Ref.IsAlive, "Element 2 died");
+                    Verify(_clrObjRef.IsAlive, "CLR object died");
+
+                    Verify((int)((FrameworkElement)_element1Ref.Target).Tag == 0, "Element 1 Tag is wrong value");
+                    Verify((int)((FrameworkElement)_element2Ref.Target).Tag == 0, "Element 2 Tag is wrong value");
+
+                    // Let go of first element, and force GC.
+                    MakeNull(ref _element1);
+                })
+                .CollectUntilTrue(() => !_element1Ref.IsAlive, "Timed out waiting for element 1 to die")
+                .CallFromUIThread(() =>
+                {
+                    Verify(_element2Ref.IsAlive, "Element 2 died");
+                    Verify(_clrObjRef.IsAlive, "CLR object died");
+
+                    // Cause PropertyChanged event.
+                    ClrObject2 clrObj = _clrObjRef.Target as ClrObject2;
+                    clrObj.IntProperty = 1;
+
+                    // Verify((int)((FrameworkElement)_element1Ref.Target).Tag == 1, "Element 1 Tag is wrong value");
+                    Verify((int)((FrameworkElement)_element2Ref.Target).Tag == 1, "Element 2 Tag is wrong value");
+                })
+                .CallFromUIThread(() =>
+                {
+                    // Let go of second element, and force GC.
+                    MakeNull(ref _element2);
+                })
+                .CollectUntilTrue(() => !_element2Ref.IsAlive, "Timed out waiting for element 2 to die")
+                .CollectUntilTrue(() => !_clrObjRef.IsAlive, "Timed out waiting for CLR object to die");
+
+            _asyncQueue.Run();
+        }
+        WeakReference _clrObjRef = null;
+        WeakReference _element1Ref = null;
+        WeakReference _element2Ref = null;
+        object _element1 = null;
+        object _element2 = null;
+
+
+        AutoResetEvent loadedSignal = new AutoResetEvent(false);
+        AutoResetEvent unloadedSignal = new AutoResetEvent(false);
+
+        //[TestMethod]
```

Comment:
Do [Ignore] work?  Might be better to use as it allows the test to still compile.

Reply:
or by commenting out TestMethod attribute as we do below in a couple other tests.

Reply:
I believe commenting out the TestMethod attribute should be enough to prevent it from running.